packages/framework/src/evaluate.ts

```diff
@@ -50,8 +50,17 @@ const formatJs = (js?: string): string => {
   // multiline js
   if (sanitizedJs.includes("\n")) {
     const lines = sanitizedJs.split("\n");
-    const lastLine = last(lines);
-    lines.splice(lines.length - 1, 1, `return ${String(lastLine)}`);
+
+    let lineIndex = -1;
+    for (let i = lines.length - 1; i >= 0; i--) {
+      if (lines[i].includes("(")) {
```

Comment:
This isn't going to work if there are two function calls in the same block. I think we just remove the return insertion for multiline blocks and require an explicit return value if needed. I'm pretty sure the only place where the return is required is the chart config so that will need to be updated.